wasm.rkt

```diff
@@ -29,7 +29,7 @@
   (c ::= real)
   (st ::= string)
   (e ::= binop relop unop (const c) (call i) call-indirect store load trap debug-inst
-     (get-local i) (set-local i) (tee-local i) (local i) (param i)
+     (get-local i) (set-local i) (tee-local i) (local i) (param i i ...)
```

Comment:
I realize this was my bad design, but I'm thinking we shouldn't have param be part of the normal instructions since it should only be legal immediately after function entry, and with your design change, only exactly once. If changed, I think we could wrap it into the reduction process for call instead of doing it as a context-independent instruction reduction. This could be a follow-up PR and don't think we need to worry about it for the presentation.